workload/tests/check_client.cc

```diff
@@ -341,6 +406,16 @@ START_TEST(test_workloadapi_Client_Connect_uses_stub)
 }
 END_TEST
 
+START_TEST(test_workloadapi_parseJWTSVID)
```

Comment:
Since you are working on test code, this could be an opportunity to review code comments on test cases. We could add some comments for each test case describing what is that case (maybe in the given/when/then format?)